woocommerce/woocommerce-admin
# Onboarding: Update global currency settings on address change

Fixes #3318

Previously, the store address updated the currency but this only occurred on page refresh.  This PR introduces several changes:
* Add functions to set the wc-admin currency in the current session.
* Updates the currency based on the store address input of the profiler.
* Standardize currency config properties and price formatting.

### Screenshots
<img width="533" alt="Screen Shot 2019-11-29 at 2 45 44 PM" src="https://user-images.githubusercontent.com/10561050/69849342-66b9f700-12b7-11ea-8f27-f7837ad8b637.png">

<img width="589" alt="Screen Shot 2019-11-29 at 2 45 55 PM" src="https://user-images.githubusercontent.com/10561050/69849341-66b9f700-12b7-11ea-8273-d4acf86937eb.png">

### Detailed test instructions:

1. Open the onboarding profiler.
2. Update your store country in the store address step to one of the countries listed in this PR - https://github.com/woocommerce/woocommerce-admin/pull/3110/files
3. Without a hard refresh of the page, navigate to "Business Details."
4. Make sure the revenue dropdown has the correct currency.
5. Make sure the rest of wc-admin currencies continue working as expected.
6.  Update to a currency not listed in `getCurrencyData()`.
7.  Check that the currency works as expected in wc-admin.
